src/main/java/org/folio/service/finance/rollover/LedgerRolloverService.java

```diff
@@ -32,7 +33,8 @@ public Future<LedgerFiscalYearRolloverCollection> getLedgerFyRollovers(String fy
     }
 
     public Future<List<LedgerFiscalYearRollover>> getLedgerFyRollovers(String fiscalYeaId, List<String> ledgerIds, RequestContext requestContext) {
-        String query = "toFiscalYearId==" + fiscalYeaId + AND + convertIdsToCqlQuery(ledgerIds, "ledgerId");
+        String query = String.format("rolloverType==%s AND toFiscalYearId==%s AND %s",
+          COMMIT, fiscalYeaId, convertIdsToCqlQuery(ledgerIds, "ledgerId"));
         RequestEntry requestEntry = new RequestEntry(LEDGER_ROLLOVERS_ENDPOINT)
                 .withQuery(query).withOffset(0).withLimit(ledgerIds.size());
         return restClient.get(requestEntry, LedgerFiscalYearRolloverCollection.class, requestContext)
```

Comment:
Karate test will be written instead of unit tests here, separate PR for karate will be opened in this sprin